screener/models.py

```diff
@@ -446,6 +446,9 @@ def is_married(self):
     def has_disability(self):
         return self.disabled or self.visually_impaired or self.long_term_disability
 
+    def is_elderly(self):
+        return self.age >= 60
```

Comment:
Different programs define elderly differently. Could you move this logic out of the member model. i.e. just call `member.age >= 60` in `HasUsdaElderlyDisabledDependency`.